### PR TITLE
Requirizing somemorefiles

### PR DESCRIPTION
**Pull request by antgonza**
Requirizing some files and also fixing @ElDeveloper comment about removing a comment block in the tests index.html

### CODE REVIEW

File: js/workers/MeshLoader.js [hunk 1]
```diff
@@ -2,42 +2,48 @@
...
+        return new THREE.BufferAttribute(array, origin.itemSize);
     }
 
-    postMessage({
-            status: 'completed',
-            attributes: attributes,
-    });
-}
-
-function cloneBufferAttribute(origin) {
```
Comment (iprotsyuk) [inline js/workers/MeshLoader.js:39]: It appears that this function is not used anywhere so I propose to delete it completely

Comment (antgonza) [inline js/workers/MeshLoader.js:39]: you are right, removing.

File: js/workers/MeshLoader.js [hunk 2]
```diff
@@ -2,42 +2,48 @@
...
-    var array = new Float32Array(origin.array.length);
-    array.set(origin.array);
-    return new THREE.BufferAttribute(array, origin.itemSize);
-}
\ No newline at end of file
+    return { 'onmessage': onmessage, 'cloneBufferAttribute': cloneBufferAttribute, 'readContents': readContents }
```
Comment (iprotsyuk) [inline js/workers/MeshLoader.js]: `readContents` isn't used outside this file so I wouldn't export it

**Comment by iprotsyuk** [comment]
a couple of comments on the scope of some functions

File: js/workers/MeshLoader.js (hunk 2 above)
Comment (antgonza) [inline js/workers/MeshLoader.js]: rm export.

File: js/workers/MeshLoader.js [hunk 3]
```diff
@@ -2,42 +2,42 @@
...
 
-onmessage = function(e) {
-    var blob = e.data;
-    var reader = new FileReaderSync();
-    readContents(reader.readAsArrayBuffer(blob));
-};
+define([
+    'three'
+],
+function (STLLoader) {
```
Comment (ElDeveloper) [inline js/workers/MeshLoader.js]: Here you are receiving the `THREE` object, but below we use the `STLLoader` object so you have to import both:

See this example with orbitcontrols:
https://github.com/biocore/emperor/blob/new-api/emperor/support_files/js/sceneplotview3d.js#L5

I think I may have misguided you!

**Comment by ElDeveloper** [comment]
👍

**Pull request merged**